.gitignore

```diff
@@ -1,7 +1,261 @@
-#idea files
```

Comment:
Why are you updating gitignore

Reply:
Some autogenerated files were missing which should be ignored

Reply:
reverted back to the original